Havit.Blazor.Components.Web.Bootstrap.Documentation/Pages/Components/Concepts/SettingsAndDefaults_Documentation.razor

```diff
@@ -0,0 +1,69 @@
+﻿@page "/concepts/SettingsAndDefaults"
+@page "/concepts/Settings"
+@page "/concepts/Defaults"
+
+<h1>Settings and defaults</h1>
+<DocAlert Type="DocAlertType.Info">Although most components support the presented functionalities, some components were constructed without settings or defaults because it wouldn't add sufficient value.</DocAlert>
+
+<h2 id="Settings">Settings</h2>
+<p>Component settings are a way of specifying component parameters. They let you bundle parameters together. That way, component settings can be easily sent across the application and applied in multiple places without the need to send the values individually. This comes in handy for instance when storing user preferences.</p>
+
+<SectionTitle Id="SettingsDemo">Demo</SectionTitle>
```

Comment:
URL anchors should follow general rules for URLs - i.e kebab-case (lower case with dashes).
`settings-demo` is the correct form here.